udata/core/dataset/api.py

```diff
@@ -288,12 +288,13 @@ def post(self, dataset):
 @api.doc(parser=upload_parser, **common_doc)
 class UploadCommunityResources(UploadMixin, API):
```

Comment:
We have two `UploadCommunityResource` class declared, I think we should remove that one? Note that in the other case the `owner` is not set.